### DISCUSSION TITLE
Update from 4.13.0-0.okd-2023-05-22-052007 to newer version

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#1732] [Q&A]
Hi.
We updated our cluster from 4.12 to 4.13.0-0.okd-2023-05-22-052007 version. There are 3 newer versions available
<img width="582" alt="image" src="https://github.com/okd-project/okd/assets/5565897/c7ef2312-66da-4851-9c1a-a5bf24213118">

Is it safe to update to any of these 3 versions? I am a bit confused, because as you can see on the pic below, there are failed updates (`4.13.0-0.okd-2023-05-22-052007 Upgrades to`) to any of 3 newer versions

<img width="795" alt="image" src="https://github.com/okd-project/okd/assets/5565897/e0b78f47-14a2-4e9f-a258-2aa23fe99037">

Thank you

**Comment by vrutkovs** [comment]
>Is it safe to update to any of these 3 versions?

Yes, that means "CI job verifying the upgrade has passed at least once". Upgrades to 07-09 failed and this edge is not available.

Whether is it safe to upgrade your clusters - we don't have a definite answer to that. In CI we focus on AWS/GCP to have comparable results. In order to evaluate the risks you want to review why the jobs failed. Seems they fall in two categories:
* initial release could not be installed:
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-gcp-modern/1683109652897730560
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1672882660294266880
* disruption of API / services was too high:
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1683109644727226368
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1683358172103839744
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1677972941041569792
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-gcp-modern/1677972993285820416
  * https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1665293971976359936

Another good idea is checking issues/discussions first - we get the majority of issues reported in first few days are the upgrade edge is available and usually provide a workaround
